src/reflect/scala/reflect/api/Internals.scala

```diff
@@ -194,7 +194,7 @@ trait Internals { self: Universe =>
     def isFreeType(symbol: Symbol): Boolean
 
     /** This symbol cast to a free type symbol.
-     *  @throws ScalaReflectionException if `isFreeType` is false.
+     *  @throws scala.reflect.ScalaReflectionException if `isFreeType` is false.
```

Comment:
I'm wrong here.  It's actually `scala.ScalaReflectionException`.

Reply:
That feels like a bug, doesn't it? Also, Internals has nested package statements so it should be visible in any case and qualification in scaladoc should not be necessary. (Not that I know how it actually works.)

Reply:
Good point.  It's not a bug in Scaladoc, but is caused by https://github.com/scala/bug/issues/11706, which is an sbt config issue, so Scaladoc can't resolve it since `ScalaReflectionException` is defined elsewhere in scala-library.  Adding `autoAPIMappings` does resolve correctly for the reasons you suggest.  I'll revert this, then.  The warnings will get fixed with 11706.

Reply:
Oh, I see. BTW I actually meant it looks like a bug or accident that the exception is not in scala.reflect.

Reply:
Oh, that's funny.  I actually had that interpretation of your question, but concluded you were talking about the package resolution.

Reply:
Yes, I meant that too.

Reply:
Ah, ok.  You really meant "Also".